app/models/scheme.rb

```diff
@@ -5,6 +5,7 @@ class Scheme < OpenStruct
   extend GdsApi::Helpers
 
   FACET_KEYS = [:areas, :business_sizes, :locations, :sectors, :stages, :support_types]
+  WHITELISTED_AREA_CODES = ["LBO","CTY", "DIS", "UTA", "EUR", "COI", "GLA", "MTD", "WMC", "LGD", "SPE", "WAE", "LGE", "EUP"]
```

Comment:
I think this [list of area types in Publisher](https://github.com/alphagov/publisher/blob/master/app/models/area.rb#L7) may be the ones it allows users to tag schemes with - if that is what it's used for then we should use the same list here, but I haven't investigated enough to be sure if that's the case.